.docker/test_all.sh

```diff
@@ -38,6 +38,22 @@ function test_libreant_search {
     curl -fs "localhost:5000/search?q=*:*"  > /dev/null
 }
 
+function check_detatched_container {
+	c_name=$1
+	running=`docker container inspect $c_name --format={{.State.Running}}`
+	if [ "$running" == "false" ]; then
+		echo -e "\nERROR: container '$c_name' is not running." 1>&2
+		echo "Container state: " 1>&2
+		docker container inspect $c_name --format="  Status:   {{.State.Status}}" 1>&2
+		docker container inspect $c_name --format="  ExitCode: {{.State.ExitCode}}" 1>&2
+		echo "Container's log tail:" 1>&2
```

Comment:
saxon genitive is for people only, not for things.  `Container last log lines:` would be better